sekoiaio.json

```diff
@@ -0,0 +1,280 @@
+{
+    "appid": "78e7ba61-eb65-421f-9bdf-28a6d14a3deb",
+    "name": "Sekoia",
+    "description": "This app will interact with SEKOIA.IO",
+    "type": "information",
+    "product_vendor": "SEKOIA.IO",
+    "logo": "logo_sekoiaio.svg",
+    "logo_dark": "logo_sekoiaio_dark.svg",
+    "product_name": "sekoia.io",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "SEKOIA.IO",
+    "license": "Copyright (c) 2023 SEKOIA.IO",
+    "app_version": "1.0.0",
+    "utctime_updated": "2023-10-09T08:07:31.780040Z",
+    "package_name": "phantom_sekoiaio",
+    "fips_compliant": false,
+    "main_module": "sekoiaio_connector.py",
+    "min_phantom_version": "6.1.1.211",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "base_url": {
+            "description": "This's the base url to request SEKOIA API",
+            "data_type": "string",
+            "required": true,
+            "default": "https://api.sekoia.io",
+            "order": 0,
+            "name": "base_url",
+            "id": 0
+        },
+        "api_key": {
+            "description": "This's a key to get access to SEKOIA API",
+            "data_type": "password",
+            "required": true,
+            "order": 1,
+            "name": "api_key",
+            "id": 1
+        },
+        "verify_server_cert": {
+            "description": "Set true or false, to verify server ssl (Default: True)",
+            "data_type": "boolean",
+            "default": true,
+            "order": 2,
+            "name": "verify_server_cert",
+            "id": 2
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get indicator",
+            "identifier": "get_indicator",
+            "description": "All the user to get an indicator according to some criteria",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "value": {
+                    "description": "Value of the indicator",
+                    "data_type": "string",
+                    "order": 0,
+                    "name": "value"
+                },
+                "type": {
+                    "description": "Type of the indicator",
```

Comment:
Can this be a set list?

Reply:
Hello, to clarify, I am wondering if the types of indicators is a definite list of options - for example: options a, b, and c. Other types of indicators are not allowed. If that is the case, you can create a list instead of a free form text field. If not, let us know and this conversation can be resolved. Thanks

Reply:
Thanks for clarification, I understand now. No we don't have this case, It's simply a string.